modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -398,12 +417,13 @@ public class IgniteImpl implements Ignite {
 
         this.cfgStorage = new DistributedConfigurationStorage(metaStorageMgr, vaultMgr);
 
+
         clusterCfgMgr = new ConfigurationManager(
                 modules.distributed().rootKeys(),
-                modules.distributed().validators(),
                 cfgStorage,
                 modules.distributed().internalSchemaExtensions(),
-                modules.distributed().polymorphicSchemaExtensions()
+                modules.distributed().polymorphicSchemaExtensions(),
```

Comment:
looks like we need to pass the `distributedConfigurationGenerator` here. Also please fix the same issue in tests. Also, `ConfigurationManager` constructor with no generator should probably be removed and corresponding `ConfigurationRegistry` code refactored